### ISSUE TITLE
Reference to the code, DOS disassembly?

### ISSUE THREAD

**Issue by cheako** [Interkarma/daggerfall-unity#1461]
I would like to check some of the interpretation.  I know this is a big task, but I'd like to review the damage/chancetohit calculations.  Is there a memory map I could go by that might list the function entry point(s)?

**Comment by Interkarma** [comment]
This work was done primarily by @Allofich. Also @df-ferital might be able to help (although he works more on talk, texture, and reputation systems than combat).

I don't have this on my end. If Allofich is able to help, he has been tagged to this issue now and will respond if he wishes to. Some of this information is also in the Daggerfall Chronicles and may be of use. Good luck!

**Comment by cheako** [comment]
I opened fall.exe in an old version of IDA(5.0) and if I'm reading this correctly it looks like the beginning is a simple de-crypt function.

**Comment by cheako** [comment]
This looks like what's needed, I'll try it later https://www.reddit.com/r/REGames/comments/ahykua/anyone_here_familiar_with_disassembling_msdos/efcykra

**Comment by cheako** [comment]
I think I got far?  I found where ```fall.exe``` prints the run ```dagger.exe``` message and marked it as __main in IDA64(7.0).  I looked for code that did things like divide by 3, add 1, add to something else(as in https://github.com/Interkarma/daggerfall-unity/blob/6063f976dd1c512f53af61134cb9a42424933bbc/Assets/Scripts/Game/Formulas/FormulaHelper.cs#L508) and did that twice conditionally.  Though I couldn't find this equation anywhere, it could be because these scripts are not part of ```fall.exe``` and instead somewhere in the game data as something like lua.
